examples/network-tools/netcat/README.md

```diff
@@ -0,0 +1,2 @@
+# Unikraft - `netcat` port
+This is the OpenBSD netcat port for Unikraft. Netcat is a simple Unix utility which reads and writes data across network connections, using TCP or UDP protocol. It is designed to be a reliable "backend" tool that can be used directly or easily driven by other programs and scripts. At the same time, it is a feature-rich network debugging and exploration tool, since it can create almost any kind of connection you would need and has several interesting built-in capabilities.
```

Comment:
```suggestion
This is the OpenBSD `netcat` port for Unikraft.
`netcat` is a simple Unix utility which reads and writes data across network connections, using TCP or UDP protocol.
It is designed to be a reliable "backend" tool that can be used directly or easily driven by other programs and scripts.
At the same time, it is a feature-rich network debugging and exploration tool, since it can create almost any kind of connection you would need and has several interesting built-in capabilities.
```